apis/nucleus/src/components/listbox/ListBoxInline.jsx

```diff
@@ -83,7 +100,7 @@ export function ListBoxInline({ app, fieldIdentifier, stateName = '$', options =
 
   const { translator } = useContext(InstanceContext);
   const moreAlignTo = useRef();
-  const [selections] = useObjectSelections(app, model);
+  const selections = selectionsApi || useObjectSelections(app, model)[0];
```

Comment:
this breaks [Rules of Hooks](https://reactjs.org/docs/hooks-rules.html)
hooks should not be called conditionally
and will probably need a more complicated workaround

Reply:
Same as above (see my comment).